app/views/account/users/show.html.slim

```diff
@@ -0,0 +1,22 @@
+
+.row
+  .col-md-2.text-center
+    h1=  @user.username
+    - if @user.username == current_user.username
```

Comment:
Is username unique? What will be if another user with the same username come here?